app/helpers/patient_query_helper.rb

```diff
@@ -241,6 +242,8 @@ def sort(patients, order, direction)
       patients = patients.order(Arel.sql('CASE WHEN latest_assessment_at IS NULL THEN 1 ELSE 0 END, latest_assessment_at ' + dir), id: dir)
     when 'workflow'
       patients = patients.order(Arel.sql('CASE WHEN isolation THEN 1 ELSE 0 END ' + dir), id: dir)
+    when 'primary_telephone'
+      patients = patients.order(Arel.sql('CASE WHEN date_of_birth IS NULL THEN 1 ELSE 0 END, primary_telephone ' + dir), id: dir)
     end
```

Comment:
Should 
`CASE WHEN date_of_birth IS NULL THEN 1 ` be `CASE WHEN primary_telephone IS NULL THEN 1 `

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1173/commits/06d6e3f3795cc1cd428a48355ed32316ce55db08